docs/manual/java/gettingstarted/GettingStartedMaven.md

```diff
@@ -9,7 +9,7 @@ Follow these instructions to create and run your first project:
 
 ## Generate a project with the Lagom archetype
 
-To create a project, invoke `mvn archetype:generate` from the command line with the following parameters (all on one line). The `archetypeVersion` parameter specifies the Lagom version (in this example, 1.3.0).  
+To create a project, invoke `mvn archetype:generate` from the command line with the following parameters (all on one line). The `archetypeVersion` parameter specifies the Lagom version (in this example, 1.3.0, but be sure to use the [current stable release](http://www.lagomframework.com/documentation/)).  
 
 ```
 mvn archetype:generate -DarchetypeGroupId=com.lightbend.lagom -DarchetypeArtifactId=maven-archetype-lagom-java
```

Comment:
The \ doesn't work for cut and paste on windows, it acts like a return. Since we need to support Windows users, I removed the \s. I discussed this with either James or Tim, can't remember now. Even though the line doesn't fit the screen, in the HTML it seems to work fine, see:
http://www.lagomframework.com/get-started-java.html and scroll down to the Lagom section step 1. I'm assuming the doc will have the same scroll bar.